docs/install/install_everest_helm_charts.md

```diff
@@ -23,7 +23,7 @@ Here are the steps to install Percona Everest and deploy additional database nam
 2. Install Percona Everest:
 
     ```sh
-    helm install everest-core percona/everest \
+    helm install percona-everest percona/everest \
```

Comment:
We've always followed the `everest-core` name for the system installation (differentiating it with the DB namespace installation named after the namespace itself). What's the reason for changing it?

https://github.com/percona/percona-helm-charts/tree/main/charts/everest